test/components/popup.spec.tsx

```diff
@@ -192,10 +232,11 @@ describe('<Popup />', function() {
                             </div>
                         </Popup>);
 
+                    const popup = new PopupTestDriver(result as Popup);
                     return waitFor(() => {
-                        const popup = bodySelect<HTMLElement>(portalId)!;
+                        const portal = popup.root as HTMLElement;
```

Comment:
I think this constant declaration is redundant. just put `popup.root` inline in `runTest(...)`

Reply:
fixed